app/models/shipit/deploy_spec/kubernetes_discovery.rb

```diff
@@ -25,10 +25,16 @@ def discover_task_definitions
 
       private
 
+      def timeout_duration
+        duration = kube_config.fetch('timeout', '900s')
+        Duration.parse(duration).to_i if duration.present? && duration != 'false'
```

Comment:
I think it would be `false` instead of `'false'`

Reply:
`false.present? => false` I was worried someone would stick the string `'false'` into the yaml. If that's not a worry, I'll remove the second check.

Reply:
It's definitely not a worry.